docusaurus/docs/iOS/uikit/components/message.md

```diff
@@ -24,178 +23,317 @@ import ChatMessageBubbleViewContentProperties from '../../common-content/referen
     ChatMessageContentView -> ChatReactionsBubbleView
     ChatMessageContentView -> ChatMessageBubbleView
     ChatReactionsBubbleView -> ChatMessageReactionsView
-    ChatMessageReactionsView -> "ChatMessageReactionsView.ItemView"
+    ChatMessageReactionsView -> "ChatMessageReactionItemView"
 ` }</Digraph>
 
 ### Overview
 
-1. [`ChatMessageLayoutOptionsResolver`](message-layout-options-resolver) is called for each message and create an instance `ChatMessageLayoutOptions`. This object contains all information needed by the view about the message (ie. does the message contains reactions, is it coming from the same user, should this message be rendered as part of a group of messages, ...)
-1. [`ChatMessageContentView`](#chatmessagecontentview) holds the entire message view and all its sub-views
-1. [`ChatMessageBubbleView`](#chatmessagebubbleview) wraps the message content inside a bubble. Depending on the layout options, the bubble will have different borders and colors and will show or not the user profile and name
-1. `ChatReactionsBubbleView` is a wrapper for `ChatMessageReactionsView` 
-1. `ChatMessageReactionsView` is responsible for rendering all reactions attached to the message
-1. `ChatMessageReactionsView.ItemView` renders a single reaction as a stateful toggle 
+1. `ChatMessageLayoutOptionsResolver` calculates the `ChatMessageLayoutOptions` for each message.
+1. `ChatMessageLayoutOptions` contains all the information needed by the view to render the message.
+1. `ChatMessageContentView` holds the entire message view and all its sub-views.
+1. `ChatMessageBubbleView` wraps the message content inside a bubble. Depending on the layout options, the bubble will have different borders and colors and will show or not the user profile and name.
+1. `ChatReactionsBubbleView` is a wrapper for `ChatMessageReactionsView`.
+1. `ChatMessageReactionsView` is responsible for rendering all reactions attached to the message.
+1. `ChatMessageReactionItemView` renders a single reaction.
 
-## Customizing Layout Options
 
-Messages are rendered differently depending on their content. Layout options are flags that the `ChatMessageLayoutOptionsResolver` injects in each message view. You can customize how messages are grouped and displayed by using your own `ChatMessageLayoutOptionsResolver` class.
+## Basic Customizations
+
+In case your application only requires minimal changes to the message view, you can easily change the styling of the message subviews by replacing them with custom views, or do simple layout changes by customizing the message layout options resolver.
+
+### Changing the bubble view
```

Comment:
Nitpick: Should we keep the titles consistent with "Changing the Bubble View"?

Reply:
Fixed all casing on all docs for iOS